docs/data-sources/elb_active_standby_pools.md

```diff
@@ -0,0 +1,153 @@
+---
+subcategory: Dedicated Load Balance (Dedicated ELB)
+---
+
+# huaweicloud_elb_active_standby_pools
+
+Use this data source to get the list of active standby ELB pools.
+
+## Example Usage
+
+```hcl
+variable "pool_name" {}
+
+data "huaweicloud_elb_active_standby_pools" "test" {
+  name = var.pool_name
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String) Specifies the region in which to query the data source.
+  If omitted, the provider-level region will be used.
+
+* `name` - (Optional, String) Specifies the name of the active/standby backend cloud server group.
+
+* `pool_id` - (Optional, String) Specifies the ID of the active/standby backend server group.
```

Comment:
please ensure the product names are the same.
**cloud server group** or **server group**

Reply:
done